recipes-bsp/firmware-imx/firmware-imx_8.6.bb

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2012-2016 Freescale Semiconductor
+# Copyright 2017-2019 NXP
+# Copyright (C) 2018 O.S. Systems Software LTDA.
+SUMMARY = "Freescale i.MX firmware"
+DESCRIPTION = "Freescale i.MX firmware such as for the VPU"
+
+require firmware-imx-${PV}.inc
+
+SRC_URI_append = " \
+    file://sdma \
+    file://epdc \
+    file://regulatory \
+    file://hdmi \
+    file://sdma-firmware.service \
+    file://epdc-firmware.service \
+    file://regulatory-firmware.service \
+    file://hdmi-firmware.service \
+"
+
+PE = "1"
+
+inherit allarch systemd
+
+SYSTEMD_PACKAGES = "${PN}-sdma ${PN}-epdc ${PN}-regulatory ${PN}-hdmi"
+SYSTEMD_SERVICE_${PN}-sdma = "sdma-firmware.service"
+SYSTEMD_SERVICE_${PN}-epdc = "epdc-firmware.service"
+SYSTEMD_SERVICE_${PN}-regulatory = "regulatory-firmware.service"
+SYSTEMD_SERVICE_${PN}-hdmi = "hdmi-firmware.service"
+
+do_install() {
+    install -d ${D}${base_libdir}/firmware/imx
+
+    if ${@bb.utils.contains('DISTRO_FEATURES','systemd','true','false',d)}; then
```

Comment:
This mean there would be no scripts provided unless `systemd` is present?

Reply:
Yes. 
Those scripts are running via systemd service (e.g. sdma-firmware.service)

Reply:
OK, then why would we install FW files below if there are no scripts to load them on the device? 😉

Reply:
The condition was more for `.services`
But Yes, this will be corrected once we switch to `udev`.
I just started working on it. It will take a while to test in our labgrid, on different imx6/7/8 boards..or maybe thro' some CI

Reply:
With the udev rules, no dependency to systemd is need.